containers/src/Kokkos_OffsetView.hpp

```diff
@@ -41,6 +41,7 @@ template <typename iType,
 using IndexRange = Kokkos::Array<iType, 2>;
 
 using index_list_type = std::initializer_list<int64_t>;
+using range_type      = std::pair<int64_t, int64_t>;
```

Comment:
I am not sure about adding that guy.  I am surprised we defined `index_list_type` like that in the first place.